lib/board.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class Board
```

Comment:
It appears our `Board` is overworked. Let's consider which methods are actually part of the `Board` 's responsibility.